### PR TITLE
RCT/YJ/Replace pint_sum with sum function

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
This PR addresses replacing the ```pint_sum``` function with the ```sum``` function.

### CODE REVIEW

File: rct229/utils/pint_utils.py [hunk 1]
```diff
@@ -147,7 +147,7 @@ def calcq_to_str(unit_system, obj) -> str:
     return retval
 
 
-def pint_sum(qty_list, default=None):
+def sum(qty_list, default=None):
```
Comment (weilixu) [inline rct229/utils/pint_utils.py:150]: we do not need to change this function name from `pint_sum` to `sum`.
The intention was to replace the `pint_sum` with the python native `sum` function

Comment (yunjoonjung-PNNL) [reply rct229/utils/pint_utils.py:150]: Good point! Addressed.

**Review by weilixu — APPROVED** [review]

**Review by charliepnnl — APPROVED** [review]
This looks good. 
I did not test it. If you have, I am happy.

**Comment by charliepnnl** [comment]
I did check that there are no more instances of pint_sum in use.

**Comment by yunjoonjung-PNNL** [comment]
> This looks good. 
> 
> I did not test it. If you have, I am happy.

Thank you Charlie. All the rules passed and I believe this is a good sign to merge this PR.

**Pull request merged**